### ISSUE TITLE
Can't specify SERVICE NAME in my new connection

### ISSUE THREAD

**Issue by dorko20** [PyMySQL/PyMySQL#439]
Hello all,
is there any way how could I specify a SERVICE NAME in order to connect to my database? It is a must parameter and without this I am not able to connect. 
e.g. Oracle SQL Developer has this option and everything is working fine, but I would really like to use python directly. 

Thanks a lot for your help,
BR,
Michal.

**Comment by methane** [comment]
I don't know what you said.
Could you point reference page about "SERVICE NAME"?

**Comment by dorko20** [comment]
Hello,
Service Name = alias to an INSTANCE (or many instances). The main purpose of this is if you are running a cluster, the client can say "connect me to SALES.acme.com", the DBA can on the fly change the number of instances which are available to SALES.acme.com requests, or even move SALES.acme.com to a completely different database without the client needing to change any settings.
SERVICE_NAMES specifies one or more names for the database service to which this instance connects. You can specify multiple services names in order to distinguish among different uses of the same database. For example:
SERVICE_NAMES = sales.acme.com, widgetsales.acme.com

You can also use service names to identify a single service that is available from two different databases through the use of replication.

So basically the service name is the alias used when connecting to the database.

**Comment by methane** [comment]
Could you give me a link to MySQL documentation about the feature??
I don't feel what you explain is up to MySQL driver.
You can write connection manager (or connection pool) for it.

**Comment by dorko20** [comment]
Hi,
here is something which could help:

https://docs.oracle.com/cd/E17952_01/refman-5.1-en/multiple-windows-services.html
http://mysql-performance-1-find-1.blogspot.sk/2012/02/installing-multiple-mysql-on-same.html

I am quite new in Python and its interaction with MySQL, maybe you are right and this is not the best way how to do it. I am thankful for any solution.

**Comment by methane** [comment]
Connect to single host:

```py
import pymysql
conn = pymysql.connect(host='db.example.com')
```

Connect to one of cluster:
```py
import pymysql
improt random
servers = ["db1.example.com", "db2.example.com"]
conn = pymysql.connect(host=random.choice(servers))
```

**Comment by methane** [comment]
BTW, you shouldn't ask a question on github issue.
Bookmark here: https://stackoverflow.com/

**Comment by dorko20** [comment]
OK, 
thanks for help, I will ask in stackoverflow.
